package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@triplemint/eslint-plugin-eunomia",
+  "version": "0.0.0",
+  "description": "Home for all codified linting and style rules for The Agency",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/triplemint/eunomia.git"
+  },
+  "keywords": [
+    "lint",
+    "style",
+    "guide",
+    "rules"
+  ],
+  "author": "",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@thi.ng/diff": "^4.0.12",
+    "@types/estree": "^0.0.50",
+    "eslint": "^8.15.0",
+    "ts-node": "^10.7.0"
+  },
```

Comment:
Since this package is using `ts-node`, it should have a dependency on `typescript` since `pnpm` won't allow this package to use the parent package's installation of `typescript`.